dependencies/windows/install-boost.cmd

```diff
@@ -1,5 +1,14 @@
 @echo off
+
+REM Build Boost.
 cd install-boost
 R --vanilla --slave -f install-boost.R --args debug static
 R --vanilla --slave -f install-boost.R --args release static
 cd ..
+
+REM Build the Boost archive for upload to S3.
+echo --^> Packaging Boost 1.69.0 ...
```

Comment:
Do we need to do this packaging when building boost on dev machines?

Reply:
I suppose we don't, but it's convenient insofar that if we're building Boost from source, we're probably updating and will want the zipball to update to S3 anyway. Normally, one will just get the pre-built Boost from S3 with `install-dependencies.cmd`.

Reply:
👍